meta-mender-compulab/README.md

```diff
@@ -0,0 +1,42 @@
+# Mender integration for Compulab boards
+
+Supported boards:
+
+ - CL-SOM-iMX8 - NXP i.MX8 System-on-Module (https://www.compulab.com/products/computer-on-modules/cl-som-imx8-nxp-i-mx-8-system-on-module-computer/)
+
+## Build
+
+Download the source:
+
+    $ mkdir mender-compulab
+    $ cd mender-compulab
+    $ repo init \
+           -u git://source.codeaurora.org/external/imx/imx-manifest.git \
+           -m imx-4.9.88-2.0.0_ga.xml \
+           -b imx-linux-rocko
+    $ mkdir .repo/local_manifests
+    $ cd .repo/local_manifests/
+    $ wget https://raw.githubusercontent.com/mendersoftware/meta-mender-community/rocko/meta-mender-compulab/scripts/manifest-compulab-local-manifest.xml
+    $ wget https://raw.githubusercontent.com/mendersoftware/meta-mender-community/rocko/scripts/mender.xml
+    $ cd -
+    $ repo sync
+    $ cd .repo/local_manifests/
+    $ ln -sf ../../sources/meta-mender-community/meta-mender-compulab/scripts/manifest-compulab-local-manifest.xml .
+    $ ln -sf ../../sources/meta-mender-community/scripts/mender.xml .
+    $ cd -
+
+Setup environment
+
+    $ export MACHINE=cl-som-imx8
+    $ source fsl-setup-release.sh -b build
+    $ source ../sources/meta-compulab-bsp/tools/setup-compulab-env
+    $ cat ../sources/meta-mender-community/meta-mender-compulab/templates/bblayers.conf.append >> conf/bblayers.conf
+    $ sed -i 's@^\(PREFERRED_RPROVIDER_u-boot-fw-utils.*$\)@# commented by Mender # \1@' conf/local.conf
+    $ sed -i 's@^\(CORE_IMAGE_EXTRA_INSTALL.*$\)@# commented by Mender # \1@' conf/local.conf
+    $ sed -i 's@^\(IMAGE_FEATURES.*package-management.*$\)@# commented by Mender # \1@' conf/local.conf
```

Comment:
I am curious what these `sed -i` commands are removing and why this can not be in the vendor specific  `local.conf.append`

Reply:
The local.conf provided by the vendor setup-environment scripts includes imx examples, package-management and a number of other things that are beyond our scope here.

Reply:
Note that I am _not_ using the standard mender-community setup-environment here.

Reply:
> Note that I am not using the standard mender-community setup-environment here.

Aware of this, and that is why I am curios why you are modifying the functionality of the upstream BSP if the purpose is to integrate tightly with it? Defeats the purpose a bit, and might have as well done a "mender-community" environment in that case.

I know that I am nit picky here but just want to make sure that we agree/comply with some sort of policy.

Reply:
Yeah, understood. This is a bit of a conflict but we also generally only produce small demo images here (thus the inclusion of meta-mender-demo) so I thought in this case removing those bits was ok. I know as a _user_ of that BSP I would likely be removing them.  I can drop this if you think that's a better approach.

Reply:
In the "meta-mender-community" case then we are just building `core-image-base` together with meta-mender-demo which is just standard poky stuff. 

But as the purpose of integrating more tightly with a vendor BSP is to be as close as possible to the vendor BSP for this reason I think it is better to drop these.

Reply:
Hi @mirzak I updated per your suggestions.  Note that I specifically had to increase the TOTAL_SIZE_MB in this case and add https://github.com/mendersoftware/meta-mender-community/pull/46/files#diff-fa575c1f7060d27460b08cba85c7bb88R2 to overrule one of the defaults in the vendor provided local.conf file.

Reply:
Thanks, and the additional changes makes sense  since they are related to Mender working together with the BSP.